Dockerfile.api

```diff
@@ -1,10 +1,8 @@
 FROM golang:1.13.1 as builder
 
-RUN go get github.com/kardianos/govendor
-
 COPY ./vendor/vendor.json /go/src/github.com/moira-alert/moira/vendor/vendor.json
```

Comment:
This line need attention. Also, look for `vendor.json` usages elsewhere, and fix accordingly.